sass/components/_info-toast.scss

```diff
@@ -0,0 +1,65 @@
+@charset 'UTF-8';
+
+.info-toast {
+    position: fixed;
+    z-index: 99;
+    right: 16px;
+    bottom: 16px;
+    display: flex;
+    width: fit-content;
+    height: 40px;
+    align-items: center;
+    justify-content: space-between;
+    padding: 4px;
+    background: var(--center-channel-text);
+    border-radius: 4px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.12);
+    color: var(--sidebar-text);
+    font-weight: 600;
+    gap: 16px;
+    line-height: 20px;
+
+    .info-toast__icon_button {
+        color: inherit;
+    }
+
+    span {
+        padding-left: 4px;
+    }
+
+    >div {
+        display: flex;
+        align-items: center;
+
+        .Icon {
+            font-size: 16px;
+        }
+    }
+}
+
+.info-toast__undo {
+    display: flex;
+    height: 24px;
+    align-items: center;
+    padding: 6px 10px;
+    border: none;
+    margin-left: 8px;
+    background: var(--denim-button-bg);
```

Comment:
Quite a few of these colours seem specific to the Denim theme. Are there other theme variables in the spec that we should be using so that they match the user's theme?

Reply:
I was seeing `Denim` was mentioned in the Figma file (as below). That is why I used denim colors. Probably I misread the the design file. All css files are updated to not use denim colors.

<img width="177" alt="Screen Shot 2022-09-13 at 13 40 28" src="https://user-images.githubusercontent.com/37421564/189892738-47818656-4904-4622-94a7-ac7a92acc992.png">